webapi/src/main/scala/org/knora/webapi/messages/util/ConstructResponseUtilV2.scala

```diff
@@ -105,6 +111,25 @@ trait ConstructResponseUtilV2 {
     targetSchema: ApiV2Schema,
     requestingUser: UserADM
   ): Task[ReadResourcesSequenceV2]
+
+  /**
+   * Gets mappings referred to in query results [[Map[IRI, ResourceWithValueRdfData]]].
+   *
+   * @param queryResultsSeparated query results referring to mappings.
+   *
+   * @param requestingUser        the user making the request.
+   * @return the referred mappings.
+   */
+  def getMappingsFromQueryResultsSeparated(
+    queryResultsSeparated: _root_.scala.collection.immutable.Map[
+      _root_.org.knora.webapi.IRI,
+      _root_.org.knora.webapi.messages.util.ConstructResponseUtilV2.ResourceWithValueRdfData
+    ],
+    requestingUser: _root_.org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+  ): zio.Task[_root_.scala.collection.immutable.Map[
+    _root_.org.knora.webapi.IRI,
+    _root_.org.knora.webapi.messages.util.ConstructResponseUtilV2.MappingAndXSLTransformation
+  ]]
 }
```

Comment:
could you fix the ugly auto-imports here?

Reply:
Oh shoot! Thanks for pointing it out, will do.

Reply:
Fix it also in other classes: 152acd6756b73c50cd6319f40de0468763f57a8e